Improve gRPC supported platforms doc (#21361)
dotnet/AspNetCore.Docs: aspnetcore/grpc/supported-platforms.md

no-loc: [appsettings.json, "ASP.NET Core Identity", cookie, Cookie, Blazor, "Blazor Server", "Blazor WebAssembly", "Identity", "Let's Encrypt", Razor, SignalR]
uid: grpc/supported-platforms
---
# gRPC on .NET supported platforms

By [James Newton-King](https://twitter.com/jamesnk)

This article discusses the requirements and supported platforms for using gRPC with .NET.

> gRPC is designed to use HTTP/2 for some of its more advanced features. HTTP/2 isn't supported everywhere which can prevent apps using gRPC. To work around this requirement there is second wire-format that is compatible with HTTP/1.1 for sending gRPC calls between clients and servers:

Reviewer: ```suggestion
gRPC takes advantage of advanced features available in  HTTP/2. HTTP/2 isn't supported everywhere, but a second wire-format using HTTP/1.1 is available for sending gRPC calls between clients and servers:
```

Reviewer: I like the following better

```suggestion
gRPC takes advantage of advanced features available in  HTTP/2. HTTP/2 isn't supported everywhere, but a second wire-format using HTTP/1.1 is available for gRPC:
```
